.evergreen/integration-tests.sh

```diff
@@ -183,3 +183,21 @@ echo $MONGO_SHELL_CONNECTION_FLAGS
 
 `pwd`/mongodb/bin/mongo $MONGO_SHELL_CONNECTION_FLAGS --eval 'printjson(db.serverBuildInfo())' admin
 `pwd`/mongodb/bin/mongo $MONGO_SHELL_CONNECTION_FLAGS --eval 'printjson(db.isMaster())' admin
+
+touch mo-expansion.yml
+
+if [ -z "$MONGO_CRYPT_SHARED_DOWNLOAD_URL" ]; then
+  echo "There is no crypt_shared library for distro='$DISTRO' and version='$MONGODB_VERSION'".
+else
+  echo "Downloading crypt_shared package from $MONGO_CRYPT_SHARED_DOWNLOAD_URL"
+  download_and_extract_crypt_shared "$MONGO_CRYPT_SHARED_DOWNLOAD_URL" "$EXTRACT" "CRYPT_SHARED_LIB_PATH"
+  echo "CRYPT_SHARED_LIB_PATH: $CRYPT_SHARED_LIB_PATH"
+  if [ -z "$CRYPT_SHARED_LIB_PATH" ]; then
+    echo "CRYPT_SHARED_LIB_PATH must be assigned, but wasn't" 1>&2 # write to stderr"
+    exit 1
+  fi
+cat >>mo-expansion.yml <<EOT 
+CRYPT_SHARED_LIB_PATH: "$CRYPT_SHARED_LIB_PATH"
+EOT
```

Comment:
In the above, if the path contains a backslash `\`, it will generate escape sequences in the YAML. Using `jq`:

```sh
jq -n '{CRYPT_SHARED_LIB_PATH: $p}' --arg p "$CRYPT_SHARED_LIB_PATH" > mo-expansions.yml
```

Which will handle string quoting, etc. This is just me being somewhat paranoid, though.

Reply:
`jq` appears not to be available on the `windows-64-vs2017-large` host. The output path from download-mongodb.sh is run through `cygpath -m`. A backslash may not be a concern.